src/Microsoft.AspNet.DataProtection/LoggingExtensions.cs

```diff
@@ -421,13 +421,13 @@ public static bool IsInformationLevelEnabled(this ILogger logger)
         }
 
         /// <summary>
-        /// Returns a value stating whether the 'verbose' log level is enabled.
+        /// Returns a value stating whether the 'trace' log level is enabled.
         /// Returns false if the logger instance is null.
         /// </summary>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static bool IsVerboseLevelEnabled(this ILogger logger)
+        public static bool IsTraceLevelEnabled(this ILogger logger)
         {
-            return IsLogLevelEnabledCore(logger, LogLevel.Verbose);
```

Comment:
This is an api change. There is already an `IsDebugLevelEnabled` function. The only thing that needs to change here is `Verbose` to `Trace` so every level has a corresponding helper function.